website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -300,6 +300,7 @@ the ingest workflow.
             "distribution_endpoint": "{$.meta.distribution_endpoint}",
             "collection": "{$.meta.collection}",
             "duplicateHandling": "{$.meta.collection.duplicateHandling}",
+            "s3MultipartChunksizeMb": "{$.meta.collection.meta.s3MultipartChunksizeMb}",
```

Comment:
We should create a card to change our `multipart_chunksize_mb` variable name to `s3MultipartChunksizeMb` throughout our code.

Reply:
https://bugs.earthdata.nasa.gov/browse/ORCA-338